builtin/game/privileges.lua

```diff
@@ -28,6 +28,17 @@ function core.register_privilege(name, param)
 	core.registered_privileges[name] = def
 end
 
+function core.override_privilege(name, redefinition)
+	local privilege = core.registered_privileges[name]
+	if not privilege then
+		error("Attempt to override non-existent privilege " .. name)
+	end
+	for k, v in pairs(redefinition) do
+		rawset(privilege, k, v)
+	end
+	core.registered_privileges[name] = privilege
```

Comment:
You don't have to set `core.registered_privileges[name]` again, `privilege` is a table. In line 32, you don't get a copy of the table, but a reference to it.

Reply:
Resolved without changes (PR closed).